stripe/src/main/java/com/stripe/android/model/SetupIntentParams.java

```diff
@@ -0,0 +1,159 @@
+package com.stripe.android.model;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.stripe.android.utils.ObjectUtils;
+
+import java.util.AbstractMap;
+import java.util.HashMap;
+import java.util.Map;
+
+public final class SetupIntentParams implements StripeIntentParams {
+
+    static final String API_PARAM_CLIENT_SECRET = "client_secret";
```

Comment:
i notice that this is also present in `SourceParams`, should we just move the other two there and use that instead ?

Reply:
How about moving the shared API_PARAMS up to the `StripeIntentParams` interface?